src/app/cold-storage/rebalance-to-hot-wallet.ts

```diff
@@ -0,0 +1,33 @@
+import { BTC_NETWORK } from "@config"
+import { checkedToSats, checkedToTargetConfs } from "@domain/bitcoin"
+import { TxDecoder } from "@domain/bitcoin/onchain"
+import { ColdStorageService } from "@services/cold-storage"
+import { OnChainService } from "@services/lnd/onchain-service"
+
+export const rebalanceToHotWallet = async ({
+  walletName,
+  amount,
+  targetConfirmations,
+}: RebalanceToHotWalletArgs): Promise<ColdStoragePsbt | ApplicationError> => {
+  const checkedAmount = checkedToSats(amount)
+  if (checkedAmount instanceof Error) return checkedAmount
+
+  const targetConfs = checkedToTargetConfs(targetConfirmations)
+  if (targetConfs instanceof Error) return targetConfs
+
+  const coldStorageService = await ColdStorageService()
+  if (coldStorageService instanceof Error) return coldStorageService
+
+  const onChainService = OnChainService(TxDecoder(BTC_NETWORK))
+  if (onChainService instanceof Error) return onChainService
+
+  const onChainAddress = await onChainService.createOnChainAddress()
+  if (onChainAddress instanceof Error) return onChainAddress
+
+  return coldStorageService.createPsbt({
```

Comment:
I'm not sure to understand the use case of returning a psbt to the graphql client?

Reply:
if we want to automate this, we should post the PSBT to specter directly: https://docs.specter.solutions/desktop/api/ep_wallets_psbt/

Reply:
The api was not available on specter when I did the specter integration initially

Reply:
:thinking: the intention was not to automate `cold wallet` -> `hot wallet` was to give a way to create psbt to sign txs. Also added as an admin mutation because for ex bitcoin jungle is not using specter.

Should I remove the mutation? Please let me know